recipes/ncbi-ngs-sdk/build.sh

```diff
@@ -1,5 +1,24 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
 NCBI_OUTDIR=$SRC_DIR/ncbi-outdir
 
+if [[ $OSTYPE == darwin* ]]; then
+     export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
+fi
+
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc $BUILD_PREFIX/bin/gcc
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/g++
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc $BUILD_PREFIX/bin/cc
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/c++
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ar $BUILD_PREFIX/bin/ar
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
```

Comment:
This is not future proof. The compilers used by conda-build are exported as $CXX and $CC. I'm surprised this passed on OSX. The name of the compiler there should be different.

Reply:
Technically, `configure` should pick up on the $CC/$CXX settings in the environment.

Often, you can use command line overrides for make to get the right values:

```
make CC=$CC CXX=$CXX
```